packages/beacon-node/test/e2e/api/impl/beacon/node/endpoints.test.ts

```diff
@@ -0,0 +1,86 @@
+import {expect} from "chai";
+import {createBeaconConfig} from "@lodestar/config";
+import {chainConfig as chainConfigDef} from "@lodestar/config/default";
+import {Api, getClient} from "@lodestar/api/beacon";
+import {ApiError} from "@lodestar/api";
+import {LogLevel, testLogger} from "../../../../../utils/logger.js";
+import {getDevBeaconNode} from "../../../../../utils/node/beacon.js";
+import {BeaconNode} from "../../../../../../src/node/nodejs.js";
+
+describe("beacon node api", function () {
+  this.timeout("30s");
+
+  const restPort = 9596;
+  const config = createBeaconConfig(chainConfigDef, Buffer.alloc(32, 0xaa));
+  const validatorCount = 512;
```

Comment:
should consider reducing validator count to 1 or even 0, not relevant for /eth/v1/node/ API tests